src/factory.js

```diff
@@ -145,7 +145,17 @@ module.exports = ({ Component, createElement, PropTypes }) => {
 
                   propsSpy.clear()
                   stateSpy.clear()
-                  previousValue = c(stateProxy, propsProxy)
+
+                  try {
+                    previousValue = c(stateProxy, propsProxy)
+                  } catch (error) {
+                    if (error instanceof CircularComputedError) {
+                      throw error
+                    }
+                    console.log(`computed "${k}" throwed`, error)
```

Comment:
[`console.warn`](https://devdocs.io/dom/console/warn) would be more appropriate.